lib/Backend/LinearScan.cpp

```diff
@@ -3339,7 +3339,7 @@ LinearScan::KillImplicitRegs(IR::Instr *instr)
         return;
     }
 
-    if (instr->m_opcode == Js::OpCode::Yield)
+    if (instr->m_opcode == Js::OpCode::GeneratorBailInLabel)
```

Comment:
The OpCode "Yield" is removed in the IRBuilder, converted into a sequence of instructions handle both the pausing of the function and the resumption.

The GeneratorBailInLabel is the resume marker and hence the condition to check for spilling registers.